var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -17,18 +17,36 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     git = "https://github.com/pnnl/ExaGO.git"
     maintainers("ryandanehy", "cameronrutherford", "pelesh")
 
-    version("1.5.1", tag="v1.5.1", submodules=True)
-    version("1.5.0", commit="227f49573a28bdd234be5500b3733be78a958f15", submodules=True)
-    version("1.4.1", commit="ea607c685444b5f345bfdc9a59c345f0f30adde2", submodules=True)
-    version("1.4.0", commit="4f4c3fdb40b52ace2d6ba000e7f24b340ec8e886", submodules=True)
-    version("1.3.0", commit="58b039d746a6eac8e84b0afc01354cd58caec485", submodules=True)
-    version("1.2.0", commit="255a214ec747b7bdde7a6d8151c083067b4d0907", submodules=True)
-    version("1.1.2", commit="db3bb16e19c09e01402071623258dae4d13e5133", submodules=True)
-    version("1.1.1", commit="0e0a3f27604876749d47c06ec71daaca4b270df9", submodules=True)
-    version("1.1.0", commit="dc8dd85544ff1b55a64a3cbbbdf12b8a0c6fdaf6", submodules=True)
-    version("1.0.0", commit="230d7df2f384f68b952a1ea03aad41431eaad283")
-    version("0.99.2", commit="56961641f50827b3aa4c14524f2f978dc48b9ce5")
-    version("0.99.1", commit="0ae426c76651ba5a9dbcaeb95f18d1b8ba961690")
+    version(
+        "1.6.0", tag="v1.6.0", commit="159cd173572280ac0f6f094a71dcc3ebeeb34076", submodules=True
+    )
+    version(
+        "1.5.1", tag="v1.5.1", commit="84e9faf9d9dad8d851075eba26038338d90e6d3a", submodules=True
+    )
+    version(
+        "1.5.0", tag="v1.5.0", commit="227f49573a28bdd234be5500b3733be78a958f15", submodules=True
+    )
+    version(
+        "1.4.1", tag="v1.4.1", commit="ea607c685444b5f345bfdc9a59c345f0f30adde2", submodules=True
+    )
+    version(
+        "1.4.0", tag="v1.4.0", commit="4f4c3fdb40b52ace2d6ba000e7f24b340ec8e886", submodules=True
+    )
+    version(
+        "1.3.0", tag="v1.3.0", commit="58b039d746a6eac8e84b0afc01354cd58caec485", submodules=True
+    )
+    version(
+        "1.1.2", tag="v1.1.2", commit="db3bb16e19c09e01402071623258dae4d13e5133", submodules=True
+    )
+    version(
+        "1.1.1", tag="v1.1.1", commit="0e0a3f27604876749d47c06ec71daaca4b270df9", submodules=True
+    )
+    version(
+        "1.1.0", tag="v1.1.0", commit="dc8dd85544ff1b55a64a3cbbbdf12b8a0c6fdaf6", submodules=True
+    )
+    version("1.0.0", tag="v1.0.0", commit="230d7df2f384f68b952a1ea03aad41431eaad283")
+    version("0.99.2", tag="v0.99.2", commit="56961641f50827b3aa4c14524f2f978dc48b9ce5")
+    version("0.99.1", tag="v0.99.1", commit="0ae426c76651ba5a9dbcaeb95f18d1b8ba961690")
```

Comment:
Do these versions have submodules? Assuming the default is false, and so this is fine